planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_over/pull_over_parameters.hpp

```diff
@@ -0,0 +1,86 @@
+
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BEHAVIOR_PATH_PLANNER__SCENE_MODULE__PULL_OVER__PULL_OVER_PARAMETERS_HPP_
+#define BEHAVIOR_PATH_PLANNER__SCENE_MODULE__PULL_OVER__PULL_OVER_PARAMETERS_HPP_
+
+#include <string>
+
+namespace behavior_path_planner
+{
+struct PullOverParameters
+{
+  double request_length;
+  double th_arrived_distance;
+  double th_stopped_velocity;
+  double th_stopped_time;
+  double margin_from_boundary;
+  double decide_path_distance;
+  double maximum_deceleration;
+  // goal research
+  std::string search_priority;  // "efficient_path" or "close_goal"
+  bool enable_goal_research;
+  double forward_goal_search_length;
+  double backward_goal_search_length;
+  double goal_search_interval;
+  double goal_to_obstacle_margin;
+  // occupancy grid map
+  bool use_occupancy_grid;
+  double occupancy_grid_collision_check_margin;
+  double theta_size;
+  double obstacle_threshold;
+  // object recognition
+  bool use_object_recognition;
+  double object_recognition_collision_check_margin;
+  // shift path
+  bool enable_shift_parking;
+  int pull_over_sampling_num;
+  double maximum_lateral_jerk;
+  double minimum_lateral_jerk;
+  double deceleration_interval;
+  double pull_over_velocity;
+  double pull_over_minimum_velocity;
+  double after_pull_over_straight_distance;
+  double before_pull_over_straight_distance;
+  // parallel parking
+  bool enable_arc_forward_parking;
+  bool enable_arc_backward_parking;
+  double after_forward_parking_straight_distance;
+  double after_backward_parking_straight_distance;
+  double forward_parking_velocity;
+  double backward_parking_velocity;
+  double forward_parking_lane_departure_margin;
+  double backward_parking_lane_departure_margin;
+  double arc_path_interval;
+  double pull_over_max_steer_angle;
+  // hazard
+  double hazard_on_threshold_dis;
```

Comment:
dis means distance? If so, dist is better to me.

Reply:
yes, I changed them to distance, velocity simulator other parameters.
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/b47d458c9870cfcb74d01d9866159bd273ee1972

Reply:
todo: I will send PR for laucher

Reply:
I sent PR https://github.com/tier4/autoware_launch/pull/491